api/v1alpha1/mcad_types.go

```diff
@@ -51,6 +52,9 @@ type MCADSpec struct {
 	PodCreationTimeout int `json:"podCreationTimeout,omitempty"`
 	//podCreationTimeout: //int (default blank)
 
+	// ControllerResources TODO: Add details
```

Comment:
I think we could aadd details on this, something like "ControllerResources is responsible for determining what the CPU and Memory limits/requests" are